plugins/backend/catalog-backend-module-okta/config.d.ts

```diff
@@ -14,13 +14,17 @@
  * limitations under the License.
  */
 
+import { AccountConfig } from './src/types';
+
 export interface Config {
   catalog?: {
     providers?: {
       /**
        * Okta configuration
        */
-      okta?: AccountConfig[];
+      okta?: {
+        [authEnv: string]: AccountConfig[];
+      };
```

Comment:
This change is a breaking change, isn't it? I wonder how should we handle this one to hurt the existing users as little as possible

Reply:
Actually this change is incorrect, Ill fix it.

Reply:
Done

Reply:
Done